assets/src/components/headwayLines.tsx

```diff
@@ -21,32 +23,35 @@ const drawHeadwayLine = (
     return acc
   }
 
-  const [currentVehicle, ...rest] = ladderVehicles
+  const [currentLadderVehicle, ...rest] = ladderVehicles
+  const vehicle: VehicleOrGhost = currentLadderVehicle.vehicle
+  const headwaySpacing =
```

Comment:
Has this lost the feature flag check?

Reply:
I did remove the check that `LadderVehicle` used to do before passing `headwaySpacing` through. But `<Ladder>` checks the feature flag before drawing `<HeadwayLines>`, so that check was redundant.

Reply:
Ah, got it, ok.